recipe/meta.yaml

```diff
@@ -83,23 +83,16 @@ requirements:
     - pyyaml
     - yaml
 
-# test:
-# This test currently comes up as lint
-#   commands:
-#     # QGIS has no --version, and --help exists 2, so make it exit 0
-#     - qgis --help || [[ "$?" == "2" ]]
-# QGIS Python bindings in non-standard place and require PYTHONPATH
-# change (per docs). Hesistant to do so in activate/deactivate,
-# so test import in run_test.sh
-#    imports:
-#     - qgis.core
-#     - qgis.utils
```

Comment:
This is all in the external test so I removed the comments. I am not sure how to handle the imports...